PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -78,6 +78,9 @@ class AbstractConnection extends AbstractChannel
     /** @var int */
     protected $heartbeat;
 
+    /** @var float */
+    protected $last_frame;
```

Comment:
The only "code-style" comment I have is I want to start moving towards PSR-2 as a coding standard.  Can you make this variable `$lastFrame` instead?